blocks/ads-block/_index.scss

```diff
@@ -0,0 +1,10 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-ads-block {
+	&--admin {
```

Comment:
wasn't sure if this needed a sub-element. it's really a modifier of the sub-element that will only exist if admin